addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/http/ComparableResponse.java

```diff
@@ -92,6 +92,8 @@ public class ComparableResponse {
     private float numberWordsWeight = NUMBER_WORDS_WEIGHT;
     private float reflectionWeight = REFLECTION_WEIGHT;
     private float relevantKeywordsWeight = RELEVANT_KEYWORDS_WEIGHT;
+    // Added for param miner
```

Comment:
These type of comments are not necessary.

Reply:
Yep. I know that. I added it so that you could suggest me an actual good name :). Since valueSent is an entire payload, I decided to break it up into valueSent and value ( actually paramSent and value but I didn't change the valueSent to paramSent :) ). The current names can cause readability problems later on so :).